src/pages/sponsors/sponsor-cart-tab/components/edit-form/edit-form.js

```
@@ -0,0 +1,331 @@
+                variant="contained"
+                color="primary"
+                type="submit"
+                disabled={hasRateExpired}
+                size="large"
+                sx={{ minWidth: 150 }}
+              >
+                {t("general.save")}
+              </Button>
+            </Box>
+          </Box>
+          <NotesModal
```

ESLint

[error] 'NotesModal' is not defined.

(react/jsx-no-undef)